pkg/pods/pod.go

```diff
@@ -548,13 +548,13 @@ func (pod *Pod) getLaunchable(launchableStanza LaunchableStanza, runAsUser strin
 			Id:               launchableId,
 			RunAs:            runAsUser,
 			ConfigDir:        pod.EnvDir(),
-			Fetcher:          uri.DefaultFetcher,
 			RootDir:          launchableRootDir,
 			P2exec:           pod.P2exec,
 			RestartTimeout:   restartTimeout,
 			CgroupConfig:     launchableStanza.CgroupConfig,
 			CgroupConfigName: launchableStanza.LaunchableId,
 		}
+		ret.SetFetcher(uri.DefaultFetcher)
```

Comment:
I was kinda considering `hoist.NewLaunchable(...)` that takes these as args, including the fetcher. Then you wouldn't need to have `SetFetcher`, right? but I realize that `NewLaunchable` would have a huge number of args. I'm not sure if that's something we want...